### DISCUSSION TITLE
无法新建lotus-worker进程

### DISCUSSION THREAD

**Discussion by Lyu23** [filecoin-project/lotus#8648] [General Q&A]
您好，想请教一下为什么lotus-worker进程一直run不起来
<img width="380" alt="123321" src="https://user-images.githubusercontent.com/90447266/168211776-c1b9b179-533a-4ae4-be83-96ebd56aca46.png">

**Comment by Lyu23** [comment]
.lotusworker下的config.toml是否需要修改

**Reply by lavrock** [reply]
需要技术支持吗?

**Reply by lavrock** [reply]
你这是chia的p盘机啊，转做FIL了？
